.github/workflows/master_build_pr.yml

```diff
@@ -12,7 +12,14 @@ jobs:
 
   build-amd:
     if: ${{inputs.type != 'arm' }}
-    runs-on: self-hosted
+    runs-on: self-hosted-test
+    env:
+      API_KEY: ${{secrets.CONVOY_AUTH_TOKEN}}
+      CHANNEL_ID: ${{vars.SKGUPTA_CHANNEL_SLACK_ID}}
+      DEVOPS_TEAM: ${{vars.DEVOPS_TEAM_SLACK_ID}}
+      IMAGE_SCAN_API_URL: http://${{vars.CONVOY_DEV_API_ENDPOINT}}/devops/code-scan/vulnerabilities/validate
```

Comment:
Change `SLACK_CHANNEL_ID` `SLACK_TOKEN` and `CONVOY_DEV_API_ENDPOINT`

Reply:
api dev->prod,
slack-channel skgupta->convoy-alerts
token   ideally should be working with current one